integr8ly/integreatly-operator
# MGDAPI-4639 using 5M quota in A34 test

# Issue link


https://issues.redhat.com/browse/MGDAPI-4639

# What


It seems that the 20M quota no longer squeeze into 3 worker node cluster. Likely due to MGDAPI-4639. When testing quota change we need to use smaller quota hence I updated the test to use 5M quota instead.

# Verification steps




Create 100k 3 worker node cluster with RhOAM installed from master (OCM only allows RHOAM to install on 4 worker node clusters). You can use /managed-api-install-master/ pipeline for that.

Execute
LOCAL=false INSTALLATION_TYPE=managed-api TEST=A34 make test/e2e/single

However, this is so trivial that I think eye review should suffice
